examples/simple/violates-constraint/CV-FD1-bad-static.output

```diff
@@ -1,7 +1,7 @@
 File: CV-FD1-bad-static.c
-Line: 1
-Error: CV-FD1
-Description: duplicate parameters in function definition.
+Line: 3
+Error: CV-TDL6
```

Comment:
This isn't what you're supposed to do. The point of this file is it's a file that throws the FD1 error. If this example no longer throws an error, you need to construct a new example that does. And if no such example can be constructed, then you need to delete the example from the suite.